product_docs/docs/epas/16/working_with_oracle_data/02_enhanced_compatibility_features.mdx

```diff
@@ -33,6 +33,9 @@ You can install EDB Postgres Advanced Server in several ways to enable compatibi
 
 See [Configuration parameters compatible with Oracle databases](../reference/database_administrator_reference/02_summary_of_configuration_parameters/) and [Managing an EDB Postgres Advanced Server installation](../installing/linux_install_details/managing_an_advanced_server_installation/) for more information about the installation options supported by the EDB Postgres Advanced Server installers.
 
+!!! Note
+    Database link is not supported in PostgreSQL-compatible mode because it requires Oracle compatibility.
```

Comment:
Maybe we can link the wording "Database link" to a topic that contains a bit more info on what that is? 

https://www.enterprisedb.com/docs/epas/latest/reference/oracle_compatibility_reference/01_dblink_ora_functions_and_procedures/ 

or 

https://www.enterprisedb.com/docs/epas/latest/reference/oracle_compatibility_reference/01_dblink_ora_functions_and_procedures/